plugins/Name_Initials.py

```diff
@@ -30,15 +32,19 @@ def init(self, logger):
         self.errors[902] = self.def_class(item = 5010, level = 3, tags = ['name', 'fix:chair'],
             title = T_('Initial stuck to the name'))
 
-        import re
         self.ReInitColleNom  = re.compile(r"^(.*[A-Z]\.)([A-Z][a-z].*)$")
 
+        self.whitelist = set()
+        if "country" in self.father.config.options:
+            country = self.father.config.options.get("country")
+            self.whitelist = set(filter(lambda name: "." in name.replace(". ", ""), whitelist_from_nsi(country)))
```

Comment:
Updated the PR. Not sure what's the benefit from list over set here though, since the sequence of the entries doesn't matter and I don't foresee duplicates; I think here it's equivalent, isn't it?

Reply:
A list use less memory and (probably a bit) faster to loop on.

Reply:
Good to know, I'll keep it in mind because so far I usually used sets _unless_ duplicates and/or order was important. Thanks!